watertap/core/zero_order_costing.py

```diff
@@ -1963,7 +1963,8 @@ def cost_membrane(blk):
 
         blk.variable_operating_cost = pyo.Var(
             initialize=1,
-            units=blk.config.flowsheet_costing_block.base_currency,
+            units=blk.config.flowsheet_costing_block.base_currency
```

Comment:
I don't doubt that this change is necessary, but I don't understand it. Can you explain to me what `flowsheet_costing_block.base_period` is and why you want it in the denominator of this expression?

Reply:
`flowsheet_costing_block.base_period` is the time period that is defined in the zero order costing yaml file. I believe it threw a unit error in the total variable operating cost calculation since variable operating cost is defined on a per operating period basis. @adam-a-a made this change originally and contributed it to my fork.

Reply:
@tarnold17 this change assigns units of USD/year as opposed to USD (since the base period we use nominally is year)